mitosheet/src/mito/utils/actions.tsx

```diff
@@ -1113,6 +1174,27 @@ export const getActions = (
             searchTerms: ['percent', '%', 'number format', 'format'],
             tooltip: 'Format all of the selected columns as percentage. This only changes the display of the data, and does not effect the underlying dataframe.'
         },
+        [ActionEnum.Set_DateTime_Dtype]: {
+            type: 'build-time',
+            staticType: ActionEnum.Set_DateTime_Dtype,
+            iconToolbar: PercentIcon,
+            longTitle: 'Format as percentage',
+            actionFunction: () => {
+                closeOpenEditingPopups();
+
+                const selectedNumberSeriesColumnIDs = getSelectedNumberSeriesColumnIDs(gridState.selections, sheetData);
+                void mitoAPI.editChangeColumnDtype(sheetIndex, selectedNumberSeriesColumnIDs, ColumnDtypes.DATETIME, getRandomId())
+            },
+            isDisabled: () => {
+                if (!doesAnySheetExist(sheetDataArray)) {
+                    return 'There are no columns to format. Import data.'
+                }
+                
+                return getSelectedNumberSeriesColumnIDs(gridState.selections, sheetData).length > 0 ? defaultActionDisabledMessage : FORMAT_DISABLED_MESSAGE
```

Comment:
This is still the wrong disabled condition -- most often, users want to turn strings into date times.